nim_status/multiaccount.nim

```diff
@@ -49,27 +58,40 @@ proc buildAccount(privateKey: PrivateKey): Account =
 
   return acc
 
+proc deriveAddresses*(multiAcc: var MultiAccount, paths: seq[string]) =
+  for p in paths:
+    let skResult = derive(multiAcc.keySeed, KeyPath p)
+    var acc = buildAccount(cast[PrivateKey](skResult.get()))
+    acc.path = p
+    multiAcc.accounts.add(acc)
```

Comment:
What do you think about making `paths` parameter a type of `seq[KeyPath]`? 

Also, what do you think about making the `MultiAccount` param immutable (ie remove the `var`) and instead returning `seq[Account]`? Maybe it's a personal preference, but I guess I prefer not to use mutable parameters unless absolutely necessary. In that case, we are deriving accounts, not addresses, so maybe we could call it `deriveAccounts`.

In summary:
```suggestion
import
  std/sugar

proc deriveAccounts*(multiAcc: var MultiAccount, paths: seq[KeyPath]): seq[Account] =
  paths.map(path => (
      let skResult = derive(multiAcc.keySeed, KeyPath p)
      var acc = buildAccount(cast[PrivateKey](skResult.get()))
      acc.path = path
      return acc
  ))
```

Reply:
Done, thanks for suggestion

Reply:
With the caveat that i kept having some weird `wrong indentation`, will look into it further.